### ISSUE TITLE
Web Preview: Sharing buttons trigger console/security error

### ISSUE THREAD

**Issue by jeremeylduvall** [Automattic/wp-calypso#27459]
Currently, if I click on "View Site" in the Calypso sidebar, it opens up a web preview where I can view my site. Today, I was going to use that to share a post to Facebook after publishing it. However, when I clicked on the Facebook sharing icon within the web preview, it went to a blank screen with the following error:

```
Load denied by X-Frame-Options: https://www.facebook.com/sharer.php?u=https%3A%2F%2Fjeremey.blog%2Ffixing-social-networks%2F&t=Fixing%20Social%20Networks does not permit framing.
```

Trying to share to Twitter and LinkedIn result in the following error:

```
Blocked by Content Security Policy

An error occurred during a connection to twitter.com.

Firefox prevented this page from loading in this way because the page has a content security policy that disallows it.
```

Can we either fix this so it doesn't result in an error or hide these buttons somehow?

#### Steps to reproduce
1. Starting at URL: https://wordpress.com/view/SITE
2. Click on a sharing icon for a post (either Facebook, Twitter, or LinkedIn in my testing)

You should see an error message. Here's a screenshot, for example, of what happens after Facebook:

<img width="1132" alt="screen shot 2018-09-26 at 8 50 25 am" src="https://user-images.githubusercontent.com/7240478/46088246-40f00a00-c169-11e8-9417-fc63fa608d37.png">

**Comment by youbrokesomething** [comment]
Unfortunately, I can't reproduce this issue right now. Our flows have changed a lot since 2018. Please open a new issue if you find a way to reproduce this issue in our current flow.
